doc/source/tutorial/linalg.rst

```diff
@@ -886,24 +887,24 @@ needs to accept complex numbers* as input in order to work with this
 algorithm. For example, the following code computes the zeroth-order
 Bessel function applied to a matrix.
 
-    >>> from scipy import special, random, linalg
-    >>> np.random.seed(1234)
-    >>> A = random.rand(3, 3)
+    >>> from scipy import special, linalg
+    >>> rng = np.random.default_rng()
+    >>> A = rng.random((3, 3))
     >>> B = linalg.funm(A, lambda x: special.jv(0, x))
     >>> A
-    array([[ 0.19151945,  0.62210877,  0.43772774],
+    array([[ 0.19151945,  0.62210877,  0.43772774],  # random
```

Comment:
Can the `# random` directive be added at the end of the output? It looks kind of ugly here:

![Screenshot](https://user-images.githubusercontent.com/43181252/114898447-b909a280-9e2f-11eb-8ba2-ab6be17e9825.png)

Also, does the `# may vary` comment seem more readable than just `# random`? I think the former implies the *output* may vary while it is difficult to tell *what* is random in the latter case. Is it just me?

Reply:
I think that it can go at the bottom yes. Also sure I can change to `# may vary`. We can actually write why. It checks that the comment starts with a pattern. There are a few examples in the doc with things like `# may vary (signs)`. I could have `# may vary (seed)` or something else.

Reply:
> I think that it can go at the bottom yes. Also sure I can change to `# may vary`. We can actually write why. It checks that the comment starts with a pattern. There are a few examples in the doc with things like `# may vary (signs)`. I could have `# may vary (seed)` or something else.

That sounds good!

Reply:
ok then I will change them all like this.